controllers/storagecluster/storagecluster_controller.go

```diff
@@ -228,18 +228,18 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.Secret{}, builder.WithPredicates(predicate.GenerationChangedPredicate{})).
 		Owns(&routev1.Route{}).
 		Owns(&templatev1.Template{}).
+		// Using builder.OnlyMetadata as we are only interested in the presence and not getting this resource anywhere
+		Watches(
+			&extv1.CustomResourceDefinition{},
+			enqueueStorageClusterRequest,
+			builder.WithPredicates(
+				util.NamePredicate(VirtualMachineCrdName),
+				util.CrdCreateAndDeletePredicate(&r.Log, r.AvailableCrds, VirtualMachineCrdName)),
+			builder.OnlyMetadata).
```

Comment:
Consistency and paring with parenthesis
```suggestion
			builder.OnlyMetadata,
		).
```